extensions/Worker.Extensions.Http.AspNetCore.Analyzers/src/ITypeSymbolExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore
+{
+    internal static class ITypeSymbolExtensions
```

Comment:
please create a unit test for this this file and test the `IsOrDerivedFrom` method (should be done for all helper methods/extensions)

Reply:
Leaving this for another iteration - need to investigate how to best test this as we may need access to a compilation and semantic model.